### ISSUE TITLE
Primary repo contains rc release

### ISSUE THREAD

**Issue by edvgui** [srl-labs/containerlab#2325]
The primary repo for all containerlab releases now contains an rc release:
```
curl -s https://netdevops.fury.site/yum/repodata/primary.xml  | grep -e "-rc"
    <location href="files/ver_It62Q/containerlab_0.60.0-rc1_linux_arm64.rpm"/>
    <location href="files/ver_b1oXz/containerlab_0.60.0-rc1_linux_amd64.rpm"/>
```

Is this intentional?

**Comment by hellt** [comment]
what problem do you see with this? That it is installed by default when no version is provided with apt/yum install?

the install scripts will not pick up this version, since they use the git tag to figure out what is the released version vs the draft

**Comment by edvgui** [comment]
We use `dnf-automatic` to install updates automatically in the lab, so we are always using the latest release, with upgrade type `default`.  I don't know if we can filter out some releases based on the git tag with this tool.

To be honest, I don't know what is the standard for such yum repos, but I would have expected rc releases to be available in another repo than the one used for stable releases.

**Comment by hellt** [comment]
yeah it is a good idea. Not available today, though, for the containerlab apt/yum repos
